andela/travel_tool_back
# #161050094 Reply Comment from Email

#### What does this PR do?
create comment from user's email

#### Description of Task to be completed?
- Change sengrid to mailgun for email notification
- Modify new request email notification
- Modify approval email notification
- Modify comment email notification

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory
- Run the server `make start`
- Run the front-end of this application `https://github.com/andela/travel_tool_front`
- Login into the application
- Create a new request or click on an existing request and make a comment.
- You should receive an email notification.

#### Any background context you want to provide?
- Please check the .env.example to set all the necessary environmental variables
- To test replied comment through an email functionality might not work because it needs a public API link which will be available in staging.

#### What are the relevant pivotal tracker stories?
[#160954803](https://www.pivotaltracker.com/story/show/160954803)

#### Screenshots (if appropriate)
<img width="634" alt="screen shot 2018-10-15 at 9 38 29 pm" src="https://user-images.githubusercontent.com/28366272/46976954-bb55df00-d0c2-11e8-953e-c66991456ac1.png">

#### Questions:
N/A
